packages/react-components/react-utilities/src/hooks/useOnScrollOutside.ts

```diff
@@ -1,16 +1,23 @@
 import * as React from 'react';
 import { useEventCallback } from './useEventCallback';
-import type { UseOnClickOrScrollOutsideOptions } from './useOnClickOutside';
+import { UseOnClickOutsideOptions } from './useOnClickOutside';
+
+export type UseOnScrollOutsideOptions = Pick<UseOnClickOutsideOptions, 'element' | 'refs' | 'contains' | 'disabled'> & {
```

Comment:
As far as I can tell, the goal here is to use `UseOnClickOutsideOptions`, except replace the `callback` field? If so, it'd be better to use `Omit` instead of `Pick`:
```suggestion
export type UseOnScrollOutsideOptions = Omit<UseOnClickOutsideOptions, 'callback'> & {
```

Reply:
I'd personally prefer `Pick` since there's less risk of picking up any new properties in case of divergence in the future